### PR TITLE
Label has unused export properties.

### PR DESCRIPTION
**Pull request by frederikhors**

### CODE REVIEW

File: src/Label.svelte [hunk 1]
```diff
@@ -17,12 +17,6 @@
   export let fore;
   export { fore as for };
   export let id = '';
-  export let xs = '';
-  export let sm = '';
-  export let md = '';
-  export let lg = '';
-  export let xl = '';
-  export let widths = colWidths;
```
Comment (bestguy) [inline src/Label.svelte]: These are actually used below on line 30:

`let columnProp = $$props[colWidth];`

Can you configure linter to ignore these?  Is this eslint?

Comment (frederikhors) [reply src/Label.svelte]: I think it works the same also w/o definition with `export`.

**Comment by frederikhors** [comment]
Are we using `Label` component today?

I cannot find a single `"import Label.svelte"` in `src` files. Am I wrong?

Today this is still a problem with the warnings in CI env.

File: src/Label.svelte (hunk 1 above)
Comment (bestguy) [reply src/Label.svelte]: These are official props though `<Label xs={12} sm={3}>` , shouldn't we export all props?

**Comment by bestguy** [comment]
Label is not used internally but is a component we expose to support horizontal forms:
https://getbootstrap.com/docs/4.4/components/forms/#horizontal-form

**Comment by bestguy** [comment]
Hi @frederikhors , I believe this is now corrected in sveltestrap@3.3.2
This PR took a different approach, but seems to work: https://github.com/bestguy/sveltestrap/pull/123

If this looks okay would you mind closing PR? Thank you for the help on the library.

**Pull request closed without merging**